CourseStream/CourseStream.Data/Migrations/20231113164931_InitialMigration.Designer.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace CourseStream.Data.Migrations
 {
-    [DbContext(typeof(CourseStreamDBContext))]
+    [DbContext(typeof(CourseStreamDbContext))]
     [Migration("20231113164931_InitialMigration")]
```

Comment:
Looks a bit destructive. Won't this impact on existing DBs with data?

Reply:
There was a problem after merging main into Docker Support branch. I needed to change the naming because migration won't be applied